contracts/governance/Vesting/fouryear/FourYearVestingLogic.sol

```diff
@@ -358,4 +383,13 @@ contract FourYearVestingLogic is IFourYearVesting, FourYearVestingStorage, Appro
         selectors[0] = this.stakeTokensWithApproval.selector;
         return selectors;
     }
+
+    /**
+     * @notice Max iteration for vesting withdrawal to prevent out of gas issue.
+     *
+     * @return max iteration value.
+     */
+    function getMaxVestingWithdrawIterations() public pure returns (uint256) {
+        return 50;
```

Comment:
but in contrast with VestingLogic here it is a constant - why?
i also suggest to change it to querying from the Staking contract

Reply:
Fixed